backend/getAllUsers_test.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"backend/endpoints"
+	"database/sql"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"encoding/json"
+
+	"github.com/gin-gonic/gin"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestGetAllUsers tests the getAllUsers function
+func TestGetAllUsers(t *testing.T) {
+	// Set up test database connection
+	var err error
+	db, err = sql.Open("sqlite3", "shows.db")
+	if err != nil {
+		t.Fatalf("Failed to connect to database: %v", err)
+	}
+	defer db.Close()
+
+	// Clean up existing users
+	_, err = db.Exec("DELETE FROM Users")
```

Comment:
I don't like that this test deletes existing registered users. What happens if this test is run on production server? Instead of cleaning up the table beforehand, should just check for whatever inserts happen in the test, then delete those at the end